src_cpp/opennnBridge/nerlWorkerOpenNN.cpp

```diff
@@ -762,4 +796,24 @@ namespace nerlnet
         return res;
     }
 
+    std::shared_ptr<std::vector<int>> NerlWorkerOpenNN::get_distributed_system_train_labels_count()
+    {       
+         switch (_distributed_system_type)
+        {
+            case WORKER_DISTRIBUTED_SYSTEM_TYPE_FEDCLIENTWEIGHTEDAVGCLASSIFICATION: // Federated Client Weighted Average Classification
+            {
+                // TODO Ori - implement
+                // Return copy of the vector
+                // make sure - throw error if data_set doesn't exist
+                std::cout << "get_distributed_system_train_labels_count" << std::endl;
```

Comment:
remove print
Add assert if data_set does not exist, and this function was called